test/cluster/dtest/schema_management_test.py

```diff
@@ -0,0 +1,649 @@
+#
+# Copyright (C) 2015-present The Apache Software Foundation
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+
+import functools
+import logging
+import string
+import time
+from concurrent import futures
+from typing import NamedTuple
+
+import pytest
+from cassandra import AlreadyExists, ConsistencyLevel, InvalidRequest
+from cassandra.cluster import ThreadPoolExecutor
+from cassandra.concurrent import execute_concurrent_with_args
+from cassandra.query import SimpleStatement, dict_factory
+from ccmlib.scylla_cluster import ScyllaCluster
+
+from dtest_class import Tester, create_cf, create_ks, read_barrier
+from tools.assertions import assert_all, assert_invalid
+from tools.cluster_topology import generate_cluster_topology
+from tools.data import create_c1c2_table, insert_c1c2, query_c1c2, rows_to_list
+
+logger = logging.getLogger(__name__)
+
+class TestSchemaManagement(Tester):
+    def test_prepared_statements_work_after_node_restart_after_altering_schema_without_changing_columns(self):
+        ring_delay_sec = 5
+        self.cluster.set_configuration_options(values={"ring_delay_ms": ring_delay_sec * 1000})
+        self.cluster.populate(generate_cluster_topology(rack_num=3))
+        self.cluster.start(wait_other_notice=True)
```

Comment:
this seems like it could be abstracted into setup_cluster fixture/method and used like this in the whole test file
`nodes = setup_cluster(configuration_options, rack_num)`